data/secscan_model/__init__.py

```diff
@@ -24,18 +24,25 @@ def configure(self, app, instance_keys, storage):
 
         logger.info("===============================")
         logger.info(
-            "Using split secscan model: v4 whitelist `%s`", self._v4_namespace_whitelist,
+            "Using %s secscan model", "split" if self._v4_model is not None else "v2",
         )
+        if self._v4_model is not None:
+            logger.info("v4 whitelist: `%s`", self._v4_namespace_whitelist)
         logger.info("===============================")
 
         return self
 
     def perform_indexing(self, next_token=None):
         if next_token is None:
+            if self._v4_model is None:
+                return SplitScanToken("v4", None)
```

Comment:
Returning this skips us [down to `self._model.perform_indexing(None)`](https://github.com/quay/quay/pull/362/files#diff-a0d14413673a3951f56735fcda8fea49R49) on the next call.